projects/commudle-admin/src/app/feature-modules/public-community/components/speakers/speakers.component.scss

```diff
@@ -0,0 +1,9 @@
+.speakers-container {
+  nb-card {
+    margin: 0;
+    padding-top: 56px;
```

Comment:
Let's use variables defined in variables.scss for spacing and common classes for flex utilities